sprint-docs/09-demo-plan.md

```diff
@@ -0,0 +1,46 @@
+
+# Bookit Iteration #9: Demo Plan
+
+Demo date: Wednesday July 26, 2017
+
+1) User notification when booking fails (#7)
+2) Prevent users from attempting to make bookings in the past (#8)
+3) Current time indicator (yellow line) appears only when viewing today's agenda (#9)
```

Comment:
Do we want to try to demo the line ticking forward minute by minute?  I don't know how well that would translate in a demo, since it only kicks forward one a minute, it's not a continuous thing.